Releases.md

```diff
@@ -0,0 +1,43 @@
+## About Versioning
+
+JustSaying adopts [SemVer 2.0](https://semver.org/spec/v2.0.0.html), and uses [MinVer](https://github.com/adamralph/minver) to achieve this. [See here](https://github.com/adamralph/minver#how-it-works) for more information on how it works.
+
+## Process for Releasing
+
+First before creating a release, make sure that [the version number in `Directory.Build.Props`](https://github.com/justeat/JustSaying/blob/d30543fbfc3cf640835339efbe497466e230f220/Directory.Build.props#L22) matches the major and minor version you are looking to release.
```

Comment:
```suggestion
```

This paragraph can be removed completely. There is no reason to change this version number before releasing.